.changeset/fuzzy-terms-push.md

```diff
@@ -0,0 +1,35 @@
+---
+'@web/browser-logs': patch
+'@web/config-loader': patch
+'@web/dev-server': patch
+'@web/dev-server-core': patch
+'@web/dev-server-esbuild': patch
+'@web/dev-server-hmr': patch
+'@web/dev-server-import-maps': patch
+'@web/dev-server-legacy': patch
+'@web/dev-server-rollup': patch
+'@web/dev-server-storybook': patch
+'@web/parse5-utils': patch
+'@web/polyfills-loader': patch
+'@web/rollup-plugin-copy': patch
+'@web/rollup-plugin-html': patch
+'@web/rollup-plugin-import-meta-assets': patch
+'@web/rollup-plugin-polyfills-loader': patch
+'@web/test-runner': patch
+'@web/test-runner-browserstack': patch
+'@web/test-runner-chrome': patch
+'@web/test-runner-cli': patch
+'@web/test-runner-commands': patch
+'@web/test-runner-core': patch
+'@web/test-runner-coverage-v8': patch
+'@web/test-runner-junit-reporter': patch
+'@web/test-runner-mocha': patch
+'@web/test-runner-playwright': patch
+'@web/test-runner-puppeteer': patch
+'@web/test-runner-saucelabs': patch
+'@web/test-runner-selenium': patch
+'@web/test-runner-visual-regression': patch
+'@web/test-runner-webdriver': patch
```

Comment:
Coming together nicely. For all of the packages, we'll need to be making a "breaking" change for this update. For pre-1.0 packages, that means `minor` and for >1.0 packages that means `major`. It _seems_ like having all of these updated to align with that will put this PR in LGTM-land.